docs/formats/bsup.md

```diff
@@ -1,15 +1,15 @@
 ---
-weight: 2
-title: Super Binary
-heading: Super Binary Specification
+weight: 3
+title: BSUP (Super Binary)
+heading: Super Binary (BSUP) Format Specification
```

Comment:
Here's how I rationalized these changes to the weight/title/headings of these Format docs. After all the changes made in this PR, nearly all the references to the formats throughout the docs are now via their acronyms, so I figured the way they appear in the left hand nav (which is based on what's in the `title:` here) should lead with the acronym. I also included the word "Format" in the title that appears at the top of the rendered page because I figured it makes it more readable when a user lands directly on the page, whereas in the left hand nav the titles all appear under the group heading "Formats" and hence are more self-documenting there.

Reply:
(I will die on this hill.)

Reply:
@nwt: The reports of your hill-based death are greatly exaggerated. I've changed it over to the "Super Binary (BSUP)" style.